Kontent.Ai.Management/Models/Spaces/SpaceCreateRequestModel.cs

```diff
@@ -0,0 +1,21 @@
+using Newtonsoft.Json;
+
+namespace Kontent.Ai.Management.Models.Spaces;
+
+/// <summary>
+/// Represents the space create request model.
+/// </summary>
+public class SpaceCreateRequestModel
```

Comment:
We do not use Request suffix -> `SpaceCreateModel`

Reply:
Renamed all occurrences to SpaceCreateModel